pyproject.toml

```diff
@@ -96,6 +97,7 @@ cli = ["pipdeptree", "cron-descriptor"]
 athena = ["pyathena", "pyarrow", "s3fs", "botocore"]
 weaviate = ["weaviate-client"]
 pydantic = ["pydantic"]
+mssql = ["pyodbc", "pyarrow"]
```

Comment:
why we need pyarrow? I'd not include it if not really necessary. ie if we need this to just to get panda frames from `sql_client` then I'd skip it and let people install this themselves